packages/openapi3/test/return-types.test.ts

```diff
@@ -159,8 +159,8 @@ describe("openapi3: return types", () => {
     );
   });
 
-  it("defines separate responses for each status code property in return type", async () => {
-    const res = await openApiFor(
+  it("emits error if multiple properties are decorated with `@statusCode` in return type", async () => {
```

Comment:
sorry  missed that but that test should probably move to the http library as this check is from there.

Reply:
Moved both test to HTTP.